experimental/packages/otlp-transformer/src/common/internal.ts

```diff
@@ -24,10 +24,15 @@ import { InstrumentationScope } from '@opentelemetry/core';
 import { Resource as ISdkResource } from '@opentelemetry/resources';
 
 export function createResource(resource: ISdkResource): Resource {
-  return {
+  const result: Resource = {
     attributes: toAttributes(resource.attributes),
     droppedAttributesCount: 0,
   };
+
+  const schemaUrl = resource.getSchemaUrl?.();
+  if (schemaUrl && schemaUrl !== '') result.schemaUrl = schemaUrl;
+
+  return result;
```

Comment:
could you add a test for this to make sure it's added to the output?
tests are in `test/logs.test.ts`, `test/metrics.test.ts` and `test/trace.test.ts`